packages/@aws-cdk/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -246,13 +273,24 @@ export class Nodegroup extends Resource implements INodegroup {
     this.maxSize = props.maxSize ?? this.desiredSize;
     this.minSize = props.minSize ?? 1;
 
+    const DEFAULT_INSTANCE_TYPE = new InstanceType('t3.medium');
```

Comment:
I think we should make it:
```typescript
public static readonly DEFAULT_INSTANCE_TYPE = new InstanceType('t3.medium');
```

Or we should make it an array as the prop is actually an array `instanceTypes`
```typescript
public static readonly DEFAULT_INSTANCE_TYPES = [new InstanceType('t3.medium')];
```

ref: https://github.com/aws/aws-cdk/blob/8f6f9a8678496e131a43ca4c76e561d50a0a0de8/packages/%40aws-cdk/aws-docdb/lib/cluster.ts#L180